lib/praxis/plugins/praxis_mapper_plugin.rb

```diff
@@ -0,0 +1,203 @@
+require 'praxis-mapper'
+require 'singleton'
+
+require 'terminal-table'
+
+# Plugin for applications which use the 'praxis-mapper' gem.
+#
+# This plugin provides the following features:
+#   1. Sets up the PraxisMapper::IdentityMap for your application and assigns
+#      it to the Praxis::Application.instance.request.identity_map for access
```

Comment:
that's incorrect. "Praxis::Application.instance.request.identity_map" does not exist.
It is assigning it to the controller's request.identiti_map.